### PR TITLE
[Merged by Bors] - chore: change associativity of `+ᵥ` from `infixl` to `infixr`

### PR DESCRIPTION
**Pull request by jcommelin**
Zulip: https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Associativity.20rules.20for.20SMul.20and.20VAdd

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by vihdzp** [comment]
...why? This is opposite to `+` and `-` in Lean, and this change many more parentheses than it removes.

**Comment by vihdzp** [comment]
Nevermind, just found the Zulip. This isn't my domain so I trust this makes sense.

File: Mathlib/Algebra/Group/Defs.lean [hunk 1]
```diff
@@ -104,8 +104,8 @@ class SMul (M : Type u) (α : Type v) where
   but it is intended to be used for left actions. -/
   smul : M → α → α
 
-@[inherit_doc] infixl:65 " +ᵥ " => HVAdd.hVAdd
-@[inherit_doc] infixl:65 " -ᵥ " => VSub.vsub
+@[inherit_doc] infixr:65 " +ᵥ " => HVAdd.hVAdd
+@[inherit_doc] infixr:65 " -ᵥ " => VSub.vsub
```
Comment (YaelDillies) [inline Mathlib/Algebra/Group/Defs.lean:108]: I understand why you are changing `+ᵥ`, but why are you changing `-ᵥ`?

Comment (jcommelin) [reply Mathlib/Algebra/Group/Defs.lean:108]: Hmm, now that I think about it again, I agree that this seems to be mistaken.
```suggestion
@[inherit_doc] infixl:65 " -ᵥ " => VSub.vsub
```

**Comment by YaelDillies** [comment]
Now that you have reverted the `-ᵥ` change, can you check the brackets you are adding are still necessary?

**Review by YaelDillies — APPROVED** [review]
Thanks! 🚀

maintainer merge

**Comment by kim-em** [comment]
Needs a link in the PR description to the zulip discussion.

bors d+

**Pull request closed without merging**